modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/sync/SyncServerManager.kt

```diff
@@ -325,16 +327,10 @@ open class SyncServerManager @Inject constructor(
         cache.evictAll()
     }
 
-    fun cancelSupporterSubscription(subscriptionUuid: String): Single<Response<Void>> {
-        return getCacheTokenOrLogin {
-            server.cancelSubscription(addBearer(it), SupporterCancelRequest(subscriptionUuid))
-        }
-    }
-
```

Comment:
This change is not tied to the goal of this PR, I just noticed the method was not used and removed it.